app/src/main/java/com/example/habitsmasher/listeners/ClickListenerForFollowerButtons.java

```diff
@@ -0,0 +1,47 @@
+package com.example.habitsmasher.listeners;
+
+import androidx.fragment.app.Fragment;
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.navigation.NavController;
+import androidx.navigation.fragment.NavHostFragment;
+
+public class ClickListenerForFollowerButtons implements View.OnClickListener{
+    /**
+     * This listener is used for the followers button, and leads to the list of people that
+     * follow the user.
+     */
+    Fragment _fragment;
+    int _navAction;
+    boolean _isFollowersButton;
```

Comment:
private? public?

Reply:
Meant to be private, guess I forgot it tho.